src/editors/data/redux/video/selectors.js

```diff
@@ -47,7 +50,19 @@ export const getTranscriptDownloadUrl = createSelector(
 );
 
 export const videoSettings = createSelector(
-  Object.values(module.simpleSelectors),
+  [
```

Comment:
Originally I wanted to create a selector that took every simpleSelector. When videoId was added, we were getting an error because the values didn't line up anymore. I changed this selector to grab just these values since this should be all we need for saving.